pages/linux/cjxl.md

```diff
@@ -0,0 +1,16 @@
+# cjxl
+
+> Compress images to JPEG XL.
+> More information: <https://github.com/libjxl/libjxl>.
+
+- Convert an image to JPEG XL:
+
+`cjxl {{path/to/image}} {{path/to/output.jxl}}`
+
+- Maximize the quality and compression of the resulting image:
```

Comment:
Losslessly maximize…?

Reply:
I did a rewording that is probably better.